platform/log/zephyr/src/mender-log.c

```diff
@@ -33,36 +33,47 @@ mender_log_init(void) {
     return MENDER_OK;
 }
 
+/* IMPORTANT: Please note that the default size of internal Zephyr logger buffer is 1024 bytes; if we need log buffer size 
+              to be more than that there is CONFIG_LOG_BUFFER_SIZE option allowing to extend the buffer size. */
+#define LOG_BUFFER_SIZE_BYTES (128)
+
 mender_err_t
 mender_log_print(uint8_t level, const char *filename, const char *function, int line, char *format, ...) {
 
+    (void)filename;
     (void)function;
-    char log[256] = { 0 };
 
-    /* Format message */
     va_list args;
     va_start(args, format);
-    vsnprintf(log, sizeof(log), format, args);
-    va_end(args);
+
+    char log_buff[LOG_BUFFER_SIZE_BYTES];
+    int  ret = vsnprintf(log_buff, LOG_BUFFER_SIZE_BYTES, format, args);
+
```

Comment:
I would keep `va_end(args);` here, is just more tidy.

Reply:
Yes. The idea was to print log regardless of `va_end(args);`. If for whatever reason it fails we will still see the log. But maybe you are right it looks more tidy it it is next to `vsnprintf`.